dnsimple/domains.go

```diff
@@ -28,6 +28,18 @@ type Domain struct {
 	UpdatedAt    string `json:"updated_at,omitempty"`
 }
 
+// DomainListOptions specifies the optional parameters you can provide
+// to customize the DomainsService.ListDomains method.
+type DomainListOptions struct {
```

Comment:
@jodosha I'd not keep it related to the specific method name, because you can have more than one listing method (e.g. the one that skips pagination)